janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/compat/ES6Compat.java

```diff
@@ -25,9 +34,35 @@
 
     private static final IndexFeatures FEATURES = coreFeatures().setWildcardField(CUSTOM_ALL_FIELD).supportsGeoContains().build();
 
+    /**
+     * Script {@code inline} was renamed to {@code source}.
```

Comment:
We do not support ElasticSearch version less than 6 anymore in master branch. Currently only ES 6 and ES 7 is supported in master branch. I believe that you can move the below methods directly into `AbstractESCompat` and `supportsParams` methods won't be needed in such case

Reply:
Great, that will simplify things a lot...